conf/CODE.conf

```diff
@@ -0,0 +1,36 @@
+include "touchpoint.DEV.conf"
+include "touchpoint.UAT.conf"
+include "application"
+
+stage = "DEV"
+
+identity {
+  baseUri = "https://idapi.code.dev-theguardian.com"
+  production.keys=false
+  webapp.url="https://profile-origin.thegulocal.com"
```

Comment:
this basic conf works for some basic tests but some things need to be configured properly still, like all these references to thegulocal.com